solutions/standard/main.tf

```diff
@@ -0,0 +1,242 @@
+  providers = {
+    ibm = ibm.ibm-sm
+  }
+}
+
+# Create private cert to use for VPN server
+module "secrets_manager_private_certificate" {
+  depends_on             = [module.private_secret_engine]
+  source                 = "terraform-ibm-modules/secrets-manager-private-cert/ibm"
+  version                = "1.1.3"
+  cert_name              = "${var.prefix}-cts-vpn-private-cert"
+  cert_description       = "Private certificate"
+  cert_template          = var.certificate_template_name
+  cert_secrets_group_id  = module.secrets_manager_group.secret_group_id
+  cert_common_name       = var.cert_common_name
+  secrets_manager_guid   = module.secrets_manager.secrets_manager_guid
+  secrets_manager_region = var.region
+  providers = {
+    ibm = ibm.ibm-sm
+  }
+}
+
+########################################################################################################################
+# VPN
+########################################################################################################################
+
+resource "time_sleep" "wait_for_security_group" {
+  depends_on       = [module.client_to_site_sg.ibm_is_security_group]
+  create_duration  = "10s"
+  destroy_duration = "60s"
+}
+
+module "client_to_site_vpn" {
+  source                        = "terraform-ibm-modules/client-to-site-vpn/ibm"
+  version                       = "1.7.2"
+  server_cert_crn               = module.secrets_manager_private_certificate.secret_crn
+  vpn_gateway_name              = "${var.prefix}-c2s-vpn"
+  resource_group_id             = module.resource_group.resource_group_id
+  subnet_ids                    = [data.ibm_is_subnet.edge-vpn.id]
+  create_policy                 = var.create_policy
+  vpn_client_access_group_users = var.vpn_client_access_group_users
+  access_group_name             = "${var.prefix}-${var.access_group_name}"
+  secrets_manager_id            = module.secrets_manager.secrets_manager_guid
+  vpn_server_routes             = var.vpn_server_routes
+}
+
+# Security Group for Client-to-Site VPN
+module "client_to_site_sg" {
+  depends_on                   = [module.landing-zone]
+  source                       = "terraform-ibm-modules/security-group/ibm"
+  version                      = "2.6.1"
+  add_ibm_cloud_internal_rules = false
+  vpc_id                       = data.ibm_is_vpc.edge.id
+  resource_group               = module.resource_group.resource_group_id
+  security_group_name          = "client-to-site-sg"
+  security_group_rules         = [{
+                  direction = "inbound"
+                  name = "allow-ibm-inbound-1"
+                  remote = "0.0.0.0/0"
+                     udp = {
+                            port_max = 443
+                            port_min = 443
+                        }
+  }]
+  target_ids                  = [module.client_to_site_vpn.vpn_server_id]
+}
+
+data "ibm_is_subnet" "edge-vpn" {
+  name       = "${var.prefix}-edge-vpn-zone-1"
```

Comment:
will this always be the name? Even if someone changes things in the override json?

Reply:
If someone changes the name then they will have to change here too.